tutorial/8 - advanced.test.ts

```diff
@@ -0,0 +1,381 @@
+import { atom, constant, Derivable, derive, SettableDerivable } from '@skunkteam/sherlock';
+import { Map as ImmutableMap } from 'immutable';
+
+/**
+ * ** Your Turn **
+ *
+ * If you see this variable, you should do something about it. :-)
+ */
+export const __YOUR_TURN__ = {} as any;
+
+describe.skip('advanced', () => {
+    /**
+     * In the case a `Derivable` is required, but the value is immutable.
+     * You can use a `constant()`.
+     *
+     * This will create a readonly `Derivable`.
+     */
+    it('`constant`', () => {
+        /**
+         * We cast to `SettableDerivable` to trick TypeScript for this test.
+         * It can be valueable to know what a `constant()` is, though.
+         * So try and remove the `cast`, see what happens!
+         */
+        const c = constant('value') as unknown as SettableDerivable<string>;
+
+        /**
+         * ** Your Turn **
+         *
+         * What do you expect this `Derivable` to do on `.set()`, `.get()` etc?
+         */
+
+        // Remove this after taking your turn below.
```

Comment:
Heb dit niet op alle plekken toegevoegd. Ik verwacht niet dat mensen op `Run` klikken voordat ze dit ingevuld hebbben. Bovendien, we gebruiken dit sowieso niet op alle plekken, dus misschien toch verwijderen?